src/02_datatypes.py

```diff
@@ -14,8 +14,9 @@
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
-
+z = x + int(y)
 
 # Write a print statement that combines x + y into the string value 57
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+string = str(x) + y
```

Comment:
That works! But you don't have to create a new variable each time. 
You could have solved it like this as well: 
print(str(x) + y)